fem/bilinearform.hpp

```diff
@@ -533,6 +539,9 @@ class BilinearForm : public Matrix
    /// Sets diagonal policy used upon construction of the linear system
    void SetDiagonalPolicy(DiagonalPolicy policy);
 
+   /// Indicate that integrators are not owned by the BilinearForm
+   void UseExternalIntegrators() { extern_bfs = 1; };
```

Comment:
@ben-s-southworth, I think this is the method you are looking for.

Reply:
At least with respect to #1158.

Reply:
@v-dobrev Yep, https://github.com/mfem/mfem/issues/1158 is addressed here, but I don't think anything addresses the HypreParMatrices in https://github.com/mfem/mfem/issues/793, unless I missed it

Reply:
My understanding is that the discussion in #793 is still open. I don't think it was moved to #1158 -- at least that is my understanding.

In any case, I agree that this PR does not address #793.

Reply:
Correct, the discussion in #793 is outstanding and not addressed here.